### ISSUE TITLE
Playground does not load correctly on initial try

### ISSUE THREAD

**Issue by bit-pirate** [yujinrobot/kobuki_desktop#21]
When first loading a world file with models from the online model database, Gazebo can't find those resources (e.g. dae files).

However, once one adds the unknown objects from the online database manually to the world, the next loading of that world file works. I guess, that is because a local copy of the model exists now.

Maybe we have the wrong resource links in our world files?

E.g. the cinder block from the playground.world

```
<model name='cinder_block'>
       [...]
        <visual name='visual'>
          <geometry>
            <mesh>
              <uri>model://cinder_block/meshes/cinder_block.dae</uri>
            </mesh>
          </geometry>
        </visual>
        [...]
</model>
```

**Comment by corot** [comment]
mmm....  yes, but the log claims that it will download the missing models:

```
Warning [ModelDatabase.cc:335] Getting models from[http://gazebosim.org/models/]. This may take a few seconds.
```

I wonder if it's not a bug in the ModelDatabase.

**Comment by bit-pirate** [comment]
Following up here: https://bitbucket.org/osrf/gazebo/issue/806/gzclient-does-not-download-missing-models

**Comment by slivingston** [comment]
@bit-pirate, @jihoonl should this issue be closed? turtlebot/turtlebot_simulator#17 is closed. The referenced Gazebo issue is resolved. I just tried `roslaunch kobuki_gazebo kobuki_playground.launch` and found the models to load correctly on the initial try.

**Comment by jihoonl** [comment]
Yeah This issue has been resolved. Closing.
